src/client/components/Loader/Spinner.js

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Spinner({ scale, color }) {
```

Comment:
Hey Balint the Spinner is working as it should. So great work. Just maybe the name of the file component does not follow the rules from the CheatSheet fx Spinner.component.js. But other than that nice. 👍

Reply:
You are right. Thank you for the warning.
I remember I looked through the /components folder and I found  names like:
SortComponent.js
SpecialOffer.js
aboutus.component.js  /  with small "a" ....
I got confused about "proper" naming :P

Reply:
I get it. But you choose, and to be safe and not confused you can check the cheatsheet Daniel gave us and there is pretty clear. :)

Reply:
Rad it one more time to make sure ;) 
Now just fixing the component to be ready to push.

Reply:
Yes it's very ready to merge 💯